frontend/tests/unit/actions/steps/multi-step-action.spec.ts

```diff
@@ -247,4 +247,25 @@ describe('MultiStepAction', () => {
       flushPromises();
     });
   });
+  describe('emitter', () => {
+    it('emits `completed` event when step execution succesfully completed', async () => {
+      const action = new TestMultiStepAction([
+        new Step(generateStepData({ identifier: 'one' })),
+        new Step(generateStepData({ identifier: 'two' })),
+      ]);
+      const one = vi.fn();
+      const two = vi.fn();
+      const methods = { one, two };
+      const emitMock = vi.fn();
+      action.emit = emitMock;
+
+      try {
+        await action.executeSteps(methods);
+      } catch {
+        /* ignore */
```

Comment:
Why ignore? If you want to make the test fail when an error occurs, you can simply let it get thrown.

Reply:
Probably a copy/paste mistake here. Fixed it.